modules/ding_provider/ding_provider.loan.inc

```diff
@@ -106,6 +106,7 @@ class DingProviderLoan extends DingEntityBase {
       'loan_date',
       'expiry',
       'renewable',
+      'renewalStatus',
```

Comment:
Why is this in camelCase when the rest of the properties use snake_case?

Reply:
It should be snake_case. I'll change it.